lookup/web_app.py

```
@@ -54,58 +30,22 @@ def lookup():
-            'success': True,
-            'faces_found': len(face_encodings),
-            'matches': results
-        })
+        # Return the response from the gpu-worker directly to the client
+        response.raise_for_status()
+        return jsonify(response.json())
 
+    except requests.exceptions.RequestException as e:
+        # This could be a connection error, timeout, etc.
+        return jsonify({'error': f"Could not connect to the processing service: {e}"}), 503
     except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)